guides/cookbook/model-name.md

```diff
@@ -0,0 +1,25 @@
+# Model Name: singular or plural? What to choose? Why is that?
+
+- ⮐ [Cookbook](./index.md)
+
+## Model name convention (or why it was singular)
+
+If you have been working with EmberData for a while, you might remember convention about model names. It was a convention that model names should be singular. But why is that? Why not plural?
+
+There is no strict rule about model names. You was just using singular names because you had default Serializer configured in your app. It was doing all job of data normalization for you. Mainly it was singularizing `type` of response you had received from server.
```

Comment:
```suggestion
There is no longer any strict rule in EmberData governing what naming convention to use for resource types. Before, you may have been using singular names because you had default Serializer configured in your app. The default serializers assume types should be singular and dasherized, and since they do the job of data normalization for you they would singularize and dasherize the `types` received from your server.
```

Reply:
I think we should add a bit more in here around the history of why it used to be that you needed singular, dasherized: mainly its the path-on-disc convention in Ember that drove this.